salt/states/module.py

```diff
@@ -91,6 +91,23 @@
               delvol_on_destroy: 'True'
           }
 
+Another example that creates a recurring task that runs a batch file on a
+Windows system:
+
+.. code-block:: yaml
+
+    eventsviewer:
+      module.run:
+        - name: task.create_task
+        - m_name: 'events-viewer'
+        - user_name: System
+        - kwargs: {
```

Comment:
It's just an example that worked for a customer. It may work with a regular yaml dictionary.